docs/theme.js

```diff
@@ -0,0 +1,13 @@
+import { future as theme } from '../dist/themes'
+import prismStyle from 'react-syntax-highlighter/styles/prism/solarizedlight'
+import prismJsx from 'react-syntax-highlighter/languages/prism/jsx'
```

Comment:
Hmm, wonder if jsx should be enabled by default as well, seems like there's a high likelihood of using this for react/jsx

Reply:
Yes, makes sense actually!